File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.kt
class Solution {
    fun finalValueAfterOperations(operations: Array<String>): Int {
        var result = 0

        operations.forEach {
            val operation = it.replace("X", "")
            if (operation[0] == '-') {
                result--
            } else {
                result++
            }
        }

        return result
    }

}

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.go
func finalValueAfterOperations(operations []string) int {
	result := 0

	for i := 0; i < len(operations); i++ {
		var element string = operations[i]
		lhs := rune(element[0])
		rhs := rune(element[2])

		if lhs == '-' || rhs == '-' {
			result--
		} else {
			result++
		}

	}

	return result
}
